models/project-extension/src/sample-data-import/generator.ts

```diff
@@ -0,0 +1,12 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+
+import type { TSampleDataImportMetadata } from './types';
+
+const generator = Generator<TSampleDataImportMetadata>({
+  fields: {
+    completed: fake((f) => f.datatype.boolean()),
+    dataset: fake((f) => f.helpers.arrayElement(['FASHION', 'GOODSTORE'])),
```

Comment:
```suggestion
    dataset: fake((f) => f.helpers.arrayElement(['FASHION', 'GOODSTORE', 'B2B'])),
```

Reply:
[Here is the reference](https://github.com/commercetools/merchant-center-services/blob/main/packages/settings/lib/schemas/project-extension.graphql#L40).

Reply:
updated [f462170](https://github.com/commercetools/test-data/pull/562/commits/f462170b4159a79c8dd55dbfc3ef0c71a4c3a868)